src/lib/utils.ts

```diff
@@ -71,6 +71,44 @@ export const columnErrorQueryBuilder = (columns: string[]) => {
   return "select {`" + columns.join("`,`") + "`}\ntake 1"
 }
 
+export const filterQueryOperatorBuilder = (filter: string,operator:string,value:string) => {
+  // SQL
+  // return `select * from table where (${filter}) (${operator}) (${value})`
+  // PRQL
+  if(filter && operator){
```

Comment:
all these logic is duplicated here and in filterblock. You could create a utils function like this to avoid bugs and duplications:

export const getPrqlFilter = (operator: string): string => { ... }

Reply:
If I am getting this correct, is it like this
```

export const getPrqlFilter = (operator:string):string =>{
  if(operator=='equals'){
    return `==`
  }
  if(operator=='notEquals'){
    return `!=`
  }
  if(operator=='contains'){
    return `std.text.contains this.`
  }
  return ""
}

```